frontend_vue/src/components/ManageToken.vue

```diff
@@ -39,44 +39,50 @@
   </BaseMessageBox>
   <div
     v-if="manageTokenResponse"
-    class="flex flex-col justify-center p-16 md:p-32 md:mx-32 rounded-xl bg-grey-50 md:max-w-[50vw] w-full"
+    class="md:mx-32 md:max-w-[50vw] w-full"
   >
-    <Suspense>
-      <component
-        :is="dynamicComponent"
-        :token-backend-response="manageTokenResponse"
-      />
-      <template #fallback>
-        <div class="flex flex-col w-full gap-8">
-          <BaseSkeletonLoader
-            class="w-[100px]"
-            type="text"
-          />
-          <BaseSkeletonLoader
-            class="w-full"
-            type="header"
-          />
-        </div>
-      </template>
-    </Suspense>
-    <MemoDisplay class="mt-32">{{
-      manageTokenResponse.canarydrop.memo
-    }}</MemoDisplay>
-    <SettingsToken
-      :token-backend-response="manageTokenResponse"
-      class="mt-32"
-    ></SettingsToken>
-    <BaseMessageBox
-      class="mt-32"
-      :variant="hasAlerts ? 'danger' : 'info'"
-      :text-link="hasAlerts ? 'Check History' : ''"
-      @click="handleCheckHistory"
+    <div
+      class="flex flex-col justify-center p-16 md:p-32 rounded-xl bg-grey-50 md:max-w-[50vw]"
     >
-      <!-- there should no risk of injection for unescaped html -->
-      <!-- eslint-disable-next-line vue/no-v-html -->
-      <span v-html="alertsMessage"></span>
-    </BaseMessageBox>
-    <BannerDeviceCanarytools class="mb-8" />
+      <Suspense>
+        <component
+          :is="dynamicComponent"
+          :token-backend-response="manageTokenResponse"
+        />
+        <template #fallback>
+          <div class="flex flex-col w-full gap-8">
+            <BaseSkeletonLoader
+              class="w-[100px]"
+              type="text"
+            />
+            <BaseSkeletonLoader
+              class="w-full"
+              type="header"
+            />
+          </div>
+        </template>
+      </Suspense>
+      <MemoDisplay class="mt-32">{{
+        manageTokenResponse.canarydrop.memo
+      }}</MemoDisplay>
+      <SettingsToken
+        :token-backend-response="manageTokenResponse"
+        class="mt-32"
+      ></SettingsToken>
+      <BaseMessageBox
+        class="mt-32"
+        :variant="hasAlerts ? 'danger' : 'info'"
+        :text-link="hasAlerts ? 'Check History' : ''"
+        @click="handleCheckHistory"
+      >
+        <!-- there should no risk of injection for unescaped html -->
+        <!-- eslint-disable-next-line vue/no-v-html -->
+        <span v-html="alertsMessage"></span>
```

Comment:
do we really need the `v-html` tag here? looking at the alertsMessage function we have all the info we need in this component? i don't see what using v-html gets us here?

one issue you can see in the future is someone extending alertsMessage to include... token memo / something controlled by the person triggering the token

Reply:
i think we can move the html here without much fuss?

Reply:
I used it as the html comes statically from us, and the template is cleared from the logic we can nicely handle in the computed prop.
It's not dangerous for this case, but if you think it can be in the future I can rework it 🌈

Reply:
yeah i agree that it's not dangerous now but it does feel like something someone could trip over in the future

Reply:
if there was a clear reason why we can't do without it i'd maybe say we can keep it but i can't see any reason why we can't have that logic in the html?